snippets/core/whats-new/whats-new-in-30/cs/JSON.cs

```diff
@@ -16,10 +16,10 @@ public string WriteJSON()
                 Age = 18
             };
             
-            System.Console.WriteLine(System.Text.Json.Serialization.JsonSerializer.ToString(instance));
+            System.Console.WriteLine(System.Text.Json.JsonSerializer.Serialize(instance));
```

Comment:
What's the rationale for explicitly calling out the entire namespace in code, rather than adding a using directive?

For example, why do the docs explicitly state `System.Text.Json.JsonSerializer.Serialize` and `System.Console.WriteLine` rather than just `JsonSerializer.Serialize` and `Console.WriteLine`

Reply:
@ahsonkhan I find shorter snippets better for these "show someone something fast" scenarios such as the what's new article. If you have to include the `using/namespace/class` definitions, you easily increase the code shown by 10-20 lines. Bigger code snippets means more scrolling for users and possibly distracting from the point of the snippet, which is to show some small piece.

Reply:
> I find shorter snippets better for these "show someone something fast" scenarios such as the what's new article.

I see, that makes sense.

In that case, why not drop the namespace in the code. Other samples don't specify `System.` everywhere and even for the S.T.Json APIs being used, they don't specify the namespace (e.g. `JsonDocument.Parse`).

I think fully qualifying the type/API in-line, within the code snippet, makes it overly verbose and hard to read/parse out the useful details. This one in particular seems quite egregious:
```C#
System.Console.WriteLine(System.Text.Json.Serialization.JsonSerializer.ToString(instance));
```
When, all you want to highlight is the `JsonSerializer.ToString` usage, which presumably is all what the user cares about.
```C#
Console.WriteLine(JsonSerializer.ToString(instance));
```


> `using/namespace/class` definitions, you easily increase the code shown by 10-20 lines. 

You could consider just adding the important/relevant using directives (and not the namespace/class definition), even if they are commented out, as an alternative.

Reply:
I agree with @ahsonkhan. I missed that when I reviewed the article.
I think we should show code how people would do in the real world, which  typically contains the using statement, since folks will copy and paste our code and add it to their programs.